ktpinblock/build.gradle.kts

```diff
@@ -4,6 +4,6 @@ plugins {
     kotlin("multiplatform").version("1.8.21").apply(false)
 }
 
-tasks.register("clean", Delete::class) {
-    delete(rootProject.buildDir)
-}
+//tasks.register("clean", Delete::class) {
+//    delete(rootProject.buildDir)
+//}
```

Comment:
This step is very important. For some reason, `js()` cannot be added because `clean` is already there. We need to either modify the KMM plugin, or simply disable `clean` task here.